gammapy/spectrum/spectrum_extraction.py

```diff
@@ -669,13 +708,19 @@ def read_ogip(cls, dir='ogip_data'):
         obs = [SpectrumObservation.read_ogip(_) for _ in dir.glob('*.pha')]
         return cls(obs)
 
-    def to_observation_table(self):
-        """Create `~gammapy.data.ObservationTable`"""
-        names = ['OBS_ID', 'PHAFILE', 'OFFSET']
+    def to_observation_table(self, moreparameters=False):
+        """Create `~gammapy.data.ObservationTable"""
+        names = ['OBS_ID', 'PHAFILE']
         col1 = [o.obs_id for o in self]
         col2 = [o.meta.phafile for o in self]
-        col3 = [o.meta.offset.value for o in self]
-        return ObservationTable(data=[col1, col2, col3], names=names)
+        if moreparameters:
```

Comment:
If you want to you can modify this to write all info that exist in meta. something like

```
keys = ['phafile', 'muon_eff', ....]
for key in keys:
   if key in meta:
       write
```